api-report/cell.api.md

```diff
@@ -38,15 +38,17 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
     // @internal (undocumented)
     protected applyStashedOp(content: unknown): unknown;
     static create(runtime: IFluidDataStoreRuntime, id?: string): SharedCell<any>;
-    delete(): void;
+    delete(): boolean | undefined;
     empty(): boolean;
     get(): Serializable<T> | undefined;
     static getFactory(): IChannelFactory;
     protected initializeLocalCore(): void;
     // (undocumented)
     protected loadCore(storage: IChannelStorageService): Promise<void>;
     protected onDisconnect(): void;
-    protected processCore(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown): void;
+    // Warning: (ae-forgotten-export) The symbol "ICellLocalOpMetadata" needs to be exported by the entry point index.d.ts
```

Comment:
ICellLocalOpMetadata should be internal to cell. Keep the parameters "unknown" to match SharedObjectCore.